src/fprime_gds/common/testing_fw/api.py

```diff
@@ -30,15 +31,17 @@ class IntegrationTestAPI(DataHandler):
 
     NOW = "NOW"
 
-    def __init__(self, pipeline, logpath=None, fsw_order=True):
+    def __init__(self, pipeline, deployment_config, logpath=None, fsw_order=True):
```

Comment:
Making deployment_config optional
```suggestion
    def __init__(self, pipeline, deployment_config=None, logpath=None, fsw_order=True):
```

Reply:
Recommendation added.